pkgs/applications/graphics/processing/default.nix

```
@@ -52,6 +52,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-SzQemZ6iZ9o89/doV8YMv7DmyPSDyckJl3oyxJyfrm0=";
   };
 
+  postPatch = ''
+    shopt -s globstar
+    substituteInPlace **/build.xml --replace-quiet '<jar ' '<jar modificationtime="0" '
```

Use a subprocess so your shell config doesn't bleed into other steps.

```suggestion
    (
      shopt -s globstar
      substituteInPlace **/build.xml --replace-quiet '<jar ' '<jar modificationtime="0" '
    )
```